pkg/skr/skr.go

```diff
@@ -138,7 +140,22 @@ func SecureKeyRelease(identity common.Identity, certState attest.CertState, SKRK
 			return nil, errors.Wrapf(err, "could not encode RSA key as JWK")
 		}
 		return jwKey, nil
+	} else if kty == "EC-HSM" || kty == "EC" {
+		key, err := x509.ParsePKCS8PrivateKey(keyBytes)
+		if err != nil {
+			return nil, errors.Wrapf(err, "could not parse ECDSA key")
+		}
+
+		var privateEcdsaKey *ecdsa.PrivateKey = key.(*ecdsa.PrivateKey)
+
+		jwKey := jwk.NewECDSAPrivateKey()
+		err = jwKey.FromRaw(privateEcdsaKey)
+		if err != nil {
+			return nil, errors.Wrapf(err, "could not encode ECDSA key as JWK")
+		}
+		return jwKey, nil
 	} else {
-		return nil, errors.Wrapf(err, "released key type not supported")
+		errString := fmt.Sprintf("released key type %v not supported", kty)

```

Comment:
nit: why not Wrapf like the one above?

Reply:
I guess there isn't an `err` to wrap like the other case. maybe `errors.Errorf` could be used instead here?

Reply:
good point, I think Errorf would be a little cleaner

Reply:
I was following the previous standard laid out in all the other files where we create new errors.

Reply:
Also, the wrapf was causing a null reference error that Mainak's team noticed because there was no error to wrap.